fastddsspy_tool/test/application/one_shot__config.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the fastddsspy executable."""
+
+import test_class
+
+
+class TestCase_instance (test_class.TestCase):
+    """@brief A subclass of `test_class.TestCase` representing a specific test case."""
+
+    def __init__(self):
+        """
+        @brief Initialize the TestCase_instance object.
+
+        This test launch:
+            fastddsspy --config-path fastddsspy_tool/test/application/configuration/\
+                configuration_basic.yaml datareaders
+            AdvancedConfigurationExample publisher
+        """
+        super().__init__(
+            name='--configCommand',
+            one_shot=True,
+            command=[],
+            dds=False,
+            config='fastddsspy_tool/test/application/configuration/\
```

Comment:
NIT: Would have been better to have this long paths as a constants, and here just reference the constant. 
Also, in my opinion it should have been better only to set the name of the file and the path add it from the place that really knows where it is (this file does not know it)

Reply:
not applied